Makefile

```diff
@@ -47,6 +47,7 @@ shellcheck:  ## Lint shell scripts
 rust-lint: ## Lint Rust code
 	cargo fmt --check
 	cargo clippy
+	cargo clippy --features qubesdb
 
 .PHONY: rust-test
 rust-test: ## Run Rust tests
```

Comment:
We are missing a `cargo test --features qubesdb` invocation. We don't actually have any Rust tests yet, but I don't want people to not realize it later on that tests aren't being run in the same manner as clippy.

Reply:
Done in c0d5d8ec74776364597823d0c4bc5802febe1ce2.